code/modules/jobs/job_types/chaplain/chaplain_nullrod.dm

```diff
@@ -41,7 +41,10 @@
 			rods[nullrod_type] = initial(nullrod_type.menu_description)
 		//special non-nullrod subtyped shit
 		rods[/obj/item/gun/ballistic/bow/divine/with_quiver] = "A divine bow and 10 quivered holy arrows."
-		rods[/obj/item/organ/internal/cyberimp/arm/shard/scythe] = "A bargain with something sinister."
+		rods[/obj/item/organ/internal/cyberimp/arm/shard/scythe] = "A shard that implants itself into your arm, allowing you to conjure forth a vorpal scythe. \
+																Allows you to behead targets for empowered strikes. \
+																Harms you if dismiss the scythe without first causing harm to a creature. \
```

Comment:
Should be "Harms you if **you** dismiss the scythe first". It's missing that second "you".

Reply:
oops